### ISSUE TITLE
Project still active

### ISSUE THREAD

**Issue by rpahli** [drogue-iot/drogue-cloud#380]
Hello, I want to ask if this project is still active. It looks like an interesting project for us.

**Comment by ctron** [comment]
It indeed is. But I think the commit history speaks for itself. I personally don't have any more time for this project, unfortunately. I can definitely merge some PRs, it that's what you're looking for. Or, help by answering questions.

**Comment by maheshbhatiya73** [comment]
hello i can work on any task ?

**Comment by lulf** [comment]
@maheshbhatiya73 Yes, you can! Just make a note in whatever issue/task you are working on that you're doing some work on it, and/or create a new issue to track the work.
